content/about/how-it-works.md

```diff
@@ -4,15 +4,15 @@ alwaysopen: true
 weight: 1
 ---
 
-Though Workers Ecosystem behaves similar to JavaScript in the browser or in Node.JS, there are a few subtle differences in how you have to think about your code. Under the hood, the Workers Runtime uses the [V8 engine](https://v8.dev/) - the same engine used by Chromium and Node.JS. The Workers runtime also implements many of the standard [APIs](/reference/runtime/apis) available in most modern browsers.
+Though Cloudflare Workers behaves similar to JavaScript in the browser or in Node.JS, there are a few subtle differences in how you have to think about your code. Under the hood, the Workers runtime uses the [V8 engine](https://v8.dev/) - the same engine used by Chromium and Node.JS. The Workers runtime also implements many of the standard [APIs](/reference/runtime/apis) available in most modern browsers.
 
-The difference between JavaSript written for the browser or Node.JS happen at runtime. Rather than running on an individual's machine (e.g a browser application or on a centralized server), Workers functions run on [Cloudflare's Edge Network](https://www.cloudflare.com/network/) - a growing global network of thousands of machines distributed across hundreds of locations.
+The differences between JavaScript written for the browser or Node.JS happen at runtime. Rather than running on an individual's machine (e.g a browser application or on a centralized server), Workers functions run on [Cloudflare's Edge Network](https://www.cloudflare.com/network/) - a growing global network of thousands of machines distributed across hundreds of locations.
```

Comment:
```suggestion
The differences between JavaScript written for the browser or Node.js happen at runtime. Rather than running on an individual's machine (e.g a browser application or on a centralized server), Workers functions run on [Cloudflare's Edge Network](https://www.cloudflare.com/network/) - a growing global network of thousands of machines distributed across hundreds of locations.
```

While we are at it